dashboard/test/models/course_offering_test.rb

```diff
@@ -759,6 +759,20 @@ class CourseOfferingTest < ActiveSupport::TestCase
       new_course_offering.attributes.except('id', 'created_at', 'updated_at')
   end
 
+  test "can seed correct self paced pl id based on self paced pl key" do
+    course_offering = create :course_offering, key: 'course-offering-1', self_paced_pl_course_offering_id: 390
```

Comment:
this is risky because the id `390` could be taken, causing the test to behave inconsistently. what breaks if you leave this out?
```suggestion
    course_offering = create :course_offering, key: 'course-offering-1'
```

Reply:
Looks like nothing breaks if i take it out, ill make this change